README.md

```diff
@@ -36,21 +36,23 @@ is an eventually consistent process, and brief delays are expected. Until the Gi
 becomes visible, other peers will not accept connections. It is best to verify connectivity to the 
 intended nodes before executing steps that rely on them.
 
-You can do this by adding a list of targets to the action configuration:
+You can do this by adding a list of hosts to ping to the action configuration:
 
 ```yaml
 - name: Tailscale
-  uses: tailscale/github-action@v3
+  uses: tailscale/github-action@v4
   with:
-    targets: 100.x.y.z,my-machine.my-tailnet.ts.net
+    ping: 100.x.y.z,my-machine.my-tailnet.ts.net
 ```
 
-or with the [tailscale ping](https://tailscale.com/kb/1080/cli#ping) command if you do not know the targets at the time of installing Tailscale in the workflow:
+or with the [tailscale ping](https://tailscale.com/kb/1080/cli#ping) command if you do not know the peers at the time of installing Tailscale in the workflow:
 
 ```bash
 tailscale ping my-target.my-tailnet.ts.net
 ```
 
+> ⚠️ On macOS runners, one can only ping IP addresses, not hostnames.
```

Comment:
Interesting, curious what the reason / failure case is here.

Reply:
Hmm potentially I missed doing something DNS related that is required for macOS when initially adding this support 🤔 something we can follow up on since it is not related to this change though.

Reply:
I'll look into it tomorrow, would be nice to fix before cutting the v4 release.

Reply:
FWIW I was able to get a ping to succeed in a test run [here](https://github.com/tailscale/github-action/actions/runs/18179511800/job/51752473601) although there have been sporadic failures after the fact (see [here](https://github.com/tailscale/github-action/actions/runs/18200583113)).

Potentially we need to add an additional check / wait on something else to propagate for macOS specifically?

Reply:
Fixed with https://github.com/tailscale/github-action/pull/200